### DISCUSSION TITLE
Requests for Topic and Feedback on OSCAL Model Review Meeting 20221028

### DISCUSSION THREAD

**Discussion by david-waltermire** [usnistgov/OSCAL#1525] [General]
Hello everyone, we just wanted to offer community members a way to propose topics or give feedback for the OSCAL Model Review Meeting on October 28, 2022. Below is the tentative agenda.

- Review Meeting Ground Rules
- OSCAL Status Summary
- Review current active issues
- Review new issues
- Status of research on the customer responsibility matrix model (#1467)
- Open discussion

Most importantly, the team wants your feedback, fellow community member! There is a subset of GitHub issues the team maintains to highlight those issues where we most want your feedback. Please comment on them before, during, or after the meeting.

https://github.com/orgs/usnistgov/projects/25/views/11

Don't see something here or ☝️ in the list? Feel free to either post a comment here, vote up existing comments, or reply to the email in our oscal-dev@list.nist.gov mailing list ([signup instructions on our website](https://pages.nist.gov/OSCAL/contact/#oscal-mailing-lists)) with your topic ideas or ongoing feedback. Thanks in advance!

**Comment by Compton-US** [comment]
Sharing the slides I will discuss in the meeting.  There may not be time for all, but we can go as far as we wish.  Feedback is welcome on issue #1467, even if we do not discuss a slide that you would like to comment on. 

[Handout - OSCAL Model Responsibility Research 2022-10-28.pdf](https://github.com/usnistgov/OSCAL/files/9888890/Handout.-.OSCAL.Model.Responsibility.Research.2022-10-28.pdf)
